### PR TITLE
AYON: Ignore separated modules

### PR DESCRIPTION
**Pull request by iLLiCiTiT**
## Changelog Description
Do not load already separated modules from default directory.

## Additional info
The modules are not in client code when created using `create_ayon_addon.py` script, but for testing of PRs is usually copied whole openpype into local app data which causes troubles when these modules are loaded.

## Testing notes:
1. All ignored addons should be loaded in OpenPype (ftrack, shotgrid, slack, sync_server)
2. The same addons should not be loaded in AYON mode from OpenPype

### CODE REVIEW

File: openpype/modules/base.py [hunk 1]
```diff
@@ -59,6 +59,14 @@
     "example_addons",
     "default_modules",
 )
+# Modules that won't be loaded in AYON mode from "./openpype/modules"
+# - the same modules are ignored in "./server_addon/create_ayon_addons.py"
+IGNORED_FILENAMES_IN_AYON = {
```
Comment (antirotor) [inline openpype/modules/base.py:64]: not sure if this shouldn't be
```suggestion
IGNORED_MODULE_NAMES_IN_AYON = {
```

File: openpype/modules/base.py [hunk 2]
```diff
@@ -392,9 +400,9 @@ def _load_ayon_addons(openpype_modules, modules_key, log):
         folder_name = "{}_{}".format(addon_name, addon_version)
         addon_dir = os.path.join(addons_dir, folder_name)
         if not os.path.exists(addon_dir):
-            log.warning((
-                "Directory for addon {} {} does not exists. Path \"{}\""
-            ).format(addon_name, addon_version, addon_dir))
+            log.debug((
+                "Addon {} {} does not have available code."
```
Comment (antirotor) [inline openpype/modules/base.py:404]: I think previous error was more clear for me. I am not sure what the addon without available code is.

**Review by antirotor — CHANGES_REQUESTED** [review]
Otherwise it looks fine and works

File: openpype/modules/base.py (hunk 2 above)
Comment (iLLiCiTiT) [reply openpype/modules/base.py:404]:
```suggestion
                "No localized client code found for addon {} {}."
```

File: openpype/modules/base.py (hunk 1 above)
Comment (iLLiCiTiT) [reply openpype/modules/base.py:64]: Those are actually filenames.

Comment (antirotor) [reply openpype/modules/base.py:64]: ...ok

**Review by antirotor — APPROVED** [review]

**Pull request merged**